WebContentMain.cpp

```diff
@@ -1,15 +1,43 @@
+#include <AK/DeprecatedString.h>
+#include <Ladybird/FontPluginLadybird.h>
+#include <Ladybird/ImageCodecPluginLadybird.h>
 #include <LibCore/ArgsParser.h>
 #include <LibCore/EventLoop.h>
 #include <LibCore/SystemServerTakeover.h>
+#include <LibFileSystem/FileSystem.h>
 #include <LibJS/Bytecode/Interpreter.h>
 #include <LibMain/Main.h>
 #include <LibWeb/Bindings/MainThreadVM.h>
+#include <LibWeb/Loader/FrameLoader.h>
+#include <LibWeb/Platform/EventLoopPluginSerenity.h>
 #include <WebContent/ConnectionFromClient.h>
 
+DeprecatedString s_serenity_resource_root;
+
+static ErrorOr<void> platform_init() {
+    s_serenity_resource_root = TRY([]() -> ErrorOr<DeprecatedString> {
+        auto const* source_dir = getenv("SERENITY_SOURCE_DIR");
+        if (source_dir) {
+            return DeprecatedString::formatted("{}/Base", source_dir);
+        }
+        auto* home = getenv("XDG_CONFIG_HOME") ?: getenv("HOME");
+        VERIFY(home);
+        auto home_lagom = DeprecatedString::formatted("{}/.lagom", home);
+        if (FileSystem::is_directory(home_lagom))
+            return home_lagom;
+        // FIXME: Some kind of GTK resource path?
+        return Error::from_string_view("Don't know how to load resources"sv);
```

Comment:
Probably either [`g_get_user_data_dir()`](https://docs.gtk.org/glib/func.get_user_data_dir.html) (`XDG_DATA_HOME`), [`g_get_system_data_dirs()`](https://docs.gtk.org/glib/func.get_system_data_dirs.html), or [`g_get_user_config_dir()`](https://docs.gtk.org/glib/func.get_user_config_dir.html), depending on the answer to the question above.

And if they are static resources and not something that the user is expected to configure, we should look into embedding them into the executable with `GResource`, like we already do for the UI definition, and like app icons are embedded in Serenity.